### PR TITLE
Drop support for `pbench-copy-sosreports`

### PR DESCRIPTION
**Pull request by portante**

### CODE REVIEW

File: server/bin/pbench-dispatch.sh [hunk 1]
```diff
@@ -4,17 +4,15 @@
...
-#               Then the symlink in TODO is deleted: we don't want to
-#               deal with this tarball again; but if there are recover-
-#               able errors, we may keep it in TODO and try again later.
-#               Any errors are reported for possible action by an admin.
+# `pbench-dispatch` looks in all the TODO directories, checks MD5 sums and
+# creates symlinks in all the state directories for the particular environment
+# where this server runs (e.g. a production environment may want the tar balls
+# unpacked (TO-UNPACK), backed up (TO-BACKUP), and indexed (TO-INDEX), while
+# a satellite environment it may only need to prepare the tar ball to be synced
+# to the main server (TO-SYNC).  Then the symlink in TODO is deleted: we don't
```
Comment (portante) [inline server/bin/pbench-dispatch.sh:12]: Missing a closing parenthesis.

File: server/bin/pbench-dispatch.sh [hunk 2]
```diff
@@ -4,17 +4,15 @@
...
+# want to deal with this tarball again; but if there are recoverable errors, we
```
Comment (portante) [inline server/bin/pbench-dispatch.sh:13]: s/tarball/tar ball/

File: server/bin/pbench-unpack-tarballs.sh [hunk 3]
```diff
@@ -4,38 +4,17 @@
...
-# - move/copy-results also makes a symlink to each tarball it uploads
-#   in $ARCHIVE/TODO.
-
-# This script loops over the contents of $archive/TODO, verifies the md5
-# sum of each tarball, and if correct, it unpacks the tarball into
-# .../incoming/$(hostname -s)/.  If everything works, it then moves the
-# symlink from $ARCHIVE/TODO to $ARCHIVE/TO-COPY-SOS.
-
+# `pbench-unpack-tarballs` looks in all the TO-UNPACK or TO-RE-UNPACK
+# directories, unpacks tarballs, and moves the symlink from the TO-[RE-]UNPACK
```
Comment (portante) [inline server/bin/pbench-unpack-tarballs.sh:8]: s/tarballs/tar balls/

**Review by portante — COMMENTED** [review]
Ready for review.

File: server/bin/pbench-dispatch.sh [hunk 4]
```diff
@@ -2,29 +2,27 @@
...
-#               deal with this tarball again; but if there are recover-
-#               able errors, we may keep it in TODO and try again later.
-#               Any errors are reported for possible action by an admin.
+# results tar balls.
+
+# `pbench-dispatch` looks in all the TODO directories, checks MD5 sums and
+# creates symlinks in all the state directories for the particular environment
+# where this server runs, e.g. a production environment may want the tar balls
+# unpacked (TO-UNPACK), backed up (TO-BACKUP), and indexed (TO-INDEX), while
+# a satellite environment it may only need to prepare the tar ball to be synced
```
Comment (ndokos) [inline server/bin/pbench-dispatch.sh:11]: Nit: no "it"

**Review by ndokos — APPROVED** [review]

File: server/bin/pbench-dispatch.sh (hunk 4 above)
Comment (portante) [reply server/bin/pbench-dispatch.sh]: Fixed.

**Review by ndokos — APPROVED** [review]

**Pull request merged**